packages/@react-stately/collections/src/index.ts

```diff
@@ -14,3 +14,4 @@ export * from './Item';
 export * from './Section';
 export * from './types';
 export * from './useCollection';
+export * from './CollectionBuilder';
```

Comment:
had to export CollectionBuilder so it could be used in useComboBoxState, lemme know if we wanna do something different